bridge/hmd/openvr.py

```diff
@@ -0,0 +1,136 @@
+"""
+OpenVR
+======
+
+OpenVR Compatible head mounted display
+It uses a python wrapper to connect with the SDK
+
+"""
+
+from . import HMD as baseHMD
+
+import glew32 as glew
+import openvr_api as openvr
+import bridge_wrapper as bridge
+
+import ctypes
+
+from ctypes import (
+        c_float,
+        )
+
+class HMD(baseHMD):
+
+    def init_ctypes(self):
+        """  I spent a day getting access violations.. I don't know how dalai's just seems to work but this one doesn't work without explicity setting the return types... baffled.
```

Comment:
Bingo! You just found the fix for the 64 build issue I was having with Blender :)
I believe that if you try the 32 bits Blender with VIVE without this code it will work as well.